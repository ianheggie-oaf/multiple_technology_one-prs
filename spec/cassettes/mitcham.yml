---
http_interactions:
- request:
    method: get
    uri: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationSearchResults.aspx?Field=S&Period=L28&f=$P1.ETR.SEARCH.SL28&r=P1.WEBGUEST
    body:
      encoding: US-ASCII
      string: ''
    headers:
      Accept-Encoding:
      - gzip,deflate,identity
      Accept:
      - "*/*"
      User-Agent:
      - Mechanize/2.7.7 Ruby/2.5.8p224 (http://github.com/sparklemotion/mechanize/)
      Accept-Charset:
      - ISO-8859-1,utf-8;q=0.7,*;q=0.7
      Accept-Language:
      - en-us,en;q=0.5
      Host:
      - com-web.t1cloud.com
      Connection:
      - keep-alive
      Keep-Alive:
      - '300'
  response:
    status:
      code: 200
      message: OK
    headers:
      Content-Type:
      - text/html; charset=utf-8
      Transfer-Encoding:
      - chunked
      Connection:
      - keep-alive
      Vary:
      - Accept-Encoding
      Cache-Control:
      - no-cache, no-store
      Date:
      - Wed, 12 May 2021 03:23:32 GMT
      Expires:
      - "-1"
      Pragma:
      - no-cache
      Server:
      - Microsoft-IIS/10.0
      Set-Cookie:
      - ASP.NET_SessionId=c45si2f2og4jq02lzwndal55; path=/; HttpOnly
      - AWSELB=231FD16D0CB73485422FCEEB9C9F4EB8D209C590CCD0F85B8AFD27C9A0E14598524F44C8BF7B02D7830266F0D4353E06B49FC0C5FD631DBE4B4CB033AFAFCD2E50F825A96C88D20EBFBC4395AAF805BDD0BF05D2C2FF338777B4E3F52E0CE254AE05C25E6C;PATH=/;HTTPONLY
      X-Aspnet-Version:
      - 2.0.50727
      Content-Encoding:
      - gzip
      X-Cache:
      - Miss from cloudfront
      Via:
      - 1.1 001fc8341f16f646f257fefd39bd6c8d.cloudfront.net (CloudFront)
      X-Amz-Cf-Pop:
      - SYD4-C2
      X-Amz-Cf-Id:
      - eVglNt_UNVdSD47Na2b3XjJCIKzgfQTywqUn5EApE333ltbh1uvVig==
    body:
      encoding: ASCII-8BIT
      string: !binary |-
        H4sIAAAAAAAAA9w8aXPiSLKfdyLmP9Tyeqe7Y21AHDbubvuFLi4LDOIybGw4CqksZHSNDmP8Yv/7yyoJjAzYuKdn27t0Y6gzs/KqzKpEv/7y6y/f/ipdif1xR0az0LZQZyAoDRFljnO5UVHM5aS+hK7r/ZaCuGwe9ULf1MJcTm5nUGYWht6XXG6xWGQXxazrG7m+mnugs3B0WPL1OGBjsnqoZy5+pQAZnAfbcoLzHXNwZ2dn8VDa/duMYP3iW2iGFoHiX0QzXCL3FrXMUJthGx2jXjS1zTAkOlJwEKJCBUl4GcDIXDzom2U6czTzye15JpvNwX/e8276M2KTICdetXISucWRFWa1IMigcOmR80xIHsIcK/vEOs8E4dIiwYyQMINyF99yDCeAMHX1ZbykQPNNL9wcfYfvcVybQYGvrWD3WFWQu/s9Iv7ymMtWssWsbTrZuyADM8cjLhCdFMHr263r28jBNkyLA88hYRUqMsgm4czVzzOeG8D8WAtN1znPkL6PtTkszzI1TKt6BPvaTCUBrC/IwgQP/1s1iaWf937Dtve1Q3wTZlEKFVa8Pf9bodThsnJfzfZkXhXr2d6qzT+HhpEs1AZyr59Bpp7ChxJDN+/ph+l40YoQM1PXiZNJ8L+5GTbkUa/P9+V4glTFPbYi6JRbyB1pMbhs9eVySzIeW9I8316czatG05YK3RFvNAT6KfANOflkZd4edybCWOYNmX6KEyMZs6gA1iV5WBuWdH7QVEeqp4tCe1pQrboRStMCF45H5XnfroaT7gD6VR/xQ6nTLZyF01E1msjV5aQ4DCejcl5QvW5rMOwO5m1OK6r305H1OL5WvWmhXJ3UrHx/dDafqKVmd9S+m1y3H3uFYRnK3W6hymmF4ePQrgZ4pM70mnU/tQ0iVtXZeG5FE8CN7/J83dAH08JZMKxZCxjXSfWf68tpcbiYjNQ50GDHGiu80OV6k9FkCbDdhtyejQszq76QqwLA0QrBLW8MyECqVgfVpqRft/Owdk8rVPN4dBYN7KGt2cNHvB7Xhrfq1hcDgef5W8Ee3AqF4XzCNyQKj5+V6kPbesSjZjAZzQCPhsgblc6Er3SqYsDz6rhWN9Sr8YizhK4XaPkJN7XbDFbdeJABz5k+esjTNeqjQVVeqt1Wr1wdVhvRID8UgMYNxWq3WncGkSxhplH+XKuWZk1m0/rQmghdn+/KIl+g9NAoPWqU99LDWKwvBGtS64o8k5suwwvwu+X5QWtQGMJclD6UrsPHuN+6fdzqD+5bUve+dccvWz0hP13yDy2zsmj3KsuW1Hig/YUu7d8loqwvgT8Dci3AGh8A96bQzYd9dVDtdfMGkeN2SbOt/OS66Y0HD55W7C5Pu8Bs+M93cvCqUPomr4VI/w6q3QfF4uGtW2Oz3BnbnjUuqsKkVl0AzkttuTCGNnwWQC7u8g8tsZRX7vioJS6MbnEY6PXhctLnosn1MA90CxTAWAc+4VFbmVxbIGOhNb3jIqB/IPAyQBQ1ftgug0xb014ZdOIBeBrzSluW6yD7FqzhTplT/FWjNQynQ0mOBvAeFoamMj8zsT280+XqfHwt0LU+KvNna+8Bzq+NEYUR/aRwO315odx1YU2lBdRLVFb066bVAVz0a3U5Hi2CRlXgxvaDN86DnR+q97gwjKCdm9Ye9Otui5Kydl0A+CPuURiePeJrzxKrZ4UVDEr4Jvcarct1WMO8W2x6tIxH1el1t0HnBoGfS4xtlJeMjnydX7AaYf2H71pvplU8fzMen5eELgPwHfMMGE7T6/ZoPHrgJt1ZqHGx/rSMcqjlzxbQD/TgbAn2bj7pelDXvof2cDq35pLNPQ7qzXsN1q3XLabLoHMC0zu6PoPNP0rrDOjLnbtoSfAWBaHfBb2EV/VOXiqPPKc8NhatO9kA+37S7s/p2xjI+ZrA6NaUu8aYLV4aPNxParo1HQ0jnenqoMmA8uF6L1jbfronMJtEbaDF+NWoqmXAe8NeVRq49qyv+nA1XQre1J7cazYX21lT8LSlMNPtqjetVc1pbQD2uTvmF0MJ5CJoSfwcaBWBLES6qN6BHe2BveWmdfVRBRmZiCrYX+F6MmoyPJo1sLMSv2jW9OWkCjIW82gwHjVBTqtLsKGWvqhULhf8Jd81RH7UrXZXY2TghQM66XTn4+LwEWz0I9UDLU/n6M7BXi+oLCRw7/TaAmiuzvX6ZA2/X7Me9erEm1xrzC4LYC8nNRVoyIuS3a3Wd8Ca1prByk6qtSrIUqsDtJuYFf33QNNLUWuoKg9E5kZeObJIgxjn59Qxoj5X7AWs3QHw2A7zCGpyW1b5/pW65RpstCQ+gsxXTsSTwmkM82UA8lBu94e80pD4fuOqvZp9q/rJ/Rg1F5fFIslFPHyWz+qX0kIpVv7eywtdRcpPcjmRv5TuvEci8sqg1CrUJf7y75xXu5MMJfJDpwz1QdQ89XhDmbvjUQ/GNaOr24poXEquy8puUsZeg5VzUUTLihC3X9oRYWUj6V9NyrmkPI/Ll7Vkfj/8OyuHTomN74Vsvkvilp22APhG91ERxlfdCSv/HnVYeR6XAT9WBvw8VnaTMvaarJxLxgtxO+B3y9pzRmXUEruXd3bzVhW9scjXyo12LVhcymWRcydkfHd5cvW7+fe0eKDkRYWEMcRysW46RuYCpV8bXfcNBS9dcCNHD+LgYnN0qg92HOJn0EW6y7rrX4+PkeIapoOOj/f1odNpFg4gbAk8opnYoqFFZk/3NP47Jww87KxmZMBfmms9CsdrCq18/gb+soE3mq3HM0AwoJEgmJMlzAnxDI2BzjM8q0SXZPkFQW0cDT2FKF9ubnS3A7GEAPHDp49s7g+ruT+s5v549PHj58wF+/4th58z642rB5bS9b+y4j2tz8Uh1UZ5KTv6Ez9fnnBbTGCgu48VQP6YejRyDSB01Vz7eEGm2ZDTLDfSs1DO9bmOmoSVuRGZQkQW5EjHdz3ih8tch8vVIhKEddcmcVCWiq5+W8dhHTXLauotOTtsjDKbjLd1OryN71+SmW+mbWxGnxuR7wq9ho0NKMq2Fy6zhnm7CcSkbSswaOr6OvHPM/nMSmZnq6bcPmKBlOwg+A7m7azbUDjNglCWRsh7uiX8E10nJE7I+wTvostm1z4O5iB/JINiGOgc0eq22/FNJ9xH1f2m4bvNwPOl2sSJOoCY9Zo1ePu0dYKBhYdYGTaa2acncVhR7Mb3/BbMpphBSC1Qnruxphatqflu5K2AXDlgHgnq4CWADoOD1J2BfUG3U/0ia3NtDDZFKcOcAsu8WKsqSD4BNHIbJyMJVrvUL3UU0hln+U6nl5Xb3czFxvjVsqiEf8tRYLsASmQauv4bYEmyAJ5OAi4efRikhnMLYmsQ2vMN8BrtqgobthwD3JzkMLAqDknwBnjgyMnJ6tjQV6H0Zq7nAU4i9sNU4TVoHX6cFXm1zwznxWokokPXwKhHElmHiNrU32vjVq93o42Fl7RRJb/Tredna2OCRk4EITBcf9kjFmGHmK8KkUjPJhWRTxibTIQw6hH/3tTIfzlzizuZu2mY2BkwiPpP5PCec+hXeft07rxlbuMZdpuJjX7ByAxnQ+ybcekQiGDehzyYXCZOm1OhBcyFnib7L5es0k7J6kRTIAhquyHo1k+2GjEuCSqp0oHy1ROynYHQvurH8pVaG0QMv0emv9y7E6Wh85aVqjgcNq8k0m1Z6Dl1/6sFrLzbdBnU2VCJAd18pmg/V8gGAfFzHei8gHBHJQE5wK9Rs4OerHb4XmJFqq5vuCEEImg10V6pYuDi1RP/EEiqXKPAsq12n+5+8cCfJjkHcOCtYdH3zblv1GExoEoscER0xXTmwSoe/APhYBoM0wrVNGYhBUcju5sw8NIgE/EMkw7W6kZ1s2GlMclIxIau1v2Ncf2QXfi12+D+aPvCd+/1cJ/61usL+P7MDNCIkPmmPP4AlJS3oKSkUGI5AX8CSv3WW6jU2qJSy3XC2Y8m01twUlpbZPpTcDp9C06nWzidsmyOH40UV3oLVlxpCy2u9KfgtSvZ4qXUjC28kuyXP46X9L0mQaImQSKwKdkQb/5QmyB9r02QlDROP9AoSN9rFCRqFJ7T6QdpoPS9VkFSWtuE+lOQeoNZkKhZeI7Uj7IL0nfbBYnZhed4/TDDIH23YZCYYXiO2A7LEHuKz92MquuG1M34zZkG3te1f7G+2T3cJ3tr7aZnVieWxzs6r9sv34rFLau/b/MoD40fXvYhkyuHvdcNqwuMdc+R688hwtHIC1dM6bn7rkevPnbdpezGbz3TjNvwQWMH8mbGdbBB+vRicP/+MePeBOytiL0+Ksl/fBZcJgS50aJAflGVMoglbZ5ndDPwLLz8YrLzz+Op5WrzrwtTD2dfuHz+b19Z1uVTvLkZF72MAgSzMUSJHmSvXXgGACbdP4rpdxvbpMOB1u6a45b2UPCUxgFxO2IdEkxfm3tIcxkOmpwr5/KlXCFf4FDoIq6Qy5dZaQVokzJPOsbyO1hW7VZ27PP02djqvXIfunEr6dOA+xiGsmzc3DqVZLcIJLyu+aZ+4zmWGAWha5sBkwZayQL8NPbbYnfY9D7xAHH2Ne4H8HrxkTVVK5a8AMD+ssMI7BfhFwEw+UoBWCcfPIO7fU6KkpkPP0rZcxzALpDfjjuMSnDvEQ/7OHT959hvNOy+oKap2duX09gKzzNr5Y6vpI9jdc57D19nhMbZ7GtcWaYqvrU0eu6zMv7PW7algzH2W4inFlktwqDShTRiWdSSAxfia3Eoe1jXWbmYujF/Ow0NX08q+ngaWdgfmmSxXvmGBUuIoLmWhb2AfFl9+RpLJGDury/s2Q6guoukCdpmKNBcqrQwLPPkqLySG5Is4kN6ER+eFvEh7rdzER+PPvZcP/ygYruh04ySTQGmpxqxkxLOLn4qeoqrx1eh1H7GiS9JBbPG7wNJicQQgHQUxY3i+8BPDkLTpsdVIsweY3hPLNdjZKR17wPNjg9Y+svVdeQzqopUe8zbRETfB8b0dxeYOm68rsOuGVA813UoqXwfqCbq7YTU6Qk2FB5EgFWt0YztVS70LzaMlkNXZaWMlv7vWQBrK8R/k+S3fCWf47iTE3CTsnkuQZz+QEm/4E5WztS6aggyhSQebYxC6Nb1EW9BjJVcM2KH8ctkpSM0hAZHxzPE4jvUW5i2TTMXOq5rrerwLQmXSMC+bxIfHVPnTSW2e0+Q/GCCxkH3Xug6BF1F4TQyLbojHUEbkEJHkunQDqrr2nRgw9Fc33MBG4IUN3IMkkARZ5gWGFzFXcl+srIP5Xz+KJ8HIuTXdS0IVsN16d/NoWLCIe6EJnn4pjZHddOykOSbQJdeh+a5VBuyIqEej8r5k0KKeUgQEA+ekxmCexKw6p3CiCnjHKDVT5PH4pY85kv5Qq6QFsZCfksYJeIB++PbNgu4CVThNc21bVePt152382HIdZmYEJUbAbwIRFt/sT10rthejHF9NMiY7bm01yQmOOyJLdRvaEovZjh5XyKQC1sYMtEkmu5MAhWa5D9XP/JJqi0zfLiCXcAy1US+i7NVWbclhaEZiwbCK+tD/o0Mh3dXSDTQS0cuA7YqwW2rM9PLC8cldIcB3umEo3AjPpP4HspxfeTcq6MWmC1QmoN52AMVRfraKCqgsrLiaaXUmRqQjQGps0nC1N7PEJN4jjmLQxsYd+A2vCp8T1bgfIOkSifHCASPRemQeySMXhi8tlRsXL6fphcTlt01CH21HfnsBlZWCNIvFKu2rKClEat3kc1XgVVT7S8lCaARLN3HFQn/pT4MWtt7FCug0OnYbrhgZVb1b9bA3Cyg9unxQO4XSWOxnLB1gb86OTsXdjvkxSLi0VwaiwzmEZggEQwyBqLDw7ms2yb1hLxDrg8Ili8qfmCMf/5ynu6zU7mHD5jJ7fFzvXmvHISnxhbOSqnzbTo2p61pOZ+I+T6CYw+TTG6cgIbtTFjh5DJRi2qfKMmDNQ2qvJqK9ms06RA/RmhiY06ec9aWtmhpcXyFlsL257ZamNehQEb+npaPioVK+9BYyspRpYqPJLplUEH9t0FXr5BWWGZ4ICaOkH0tyWoYXs+hC7JjwTer9ae7fLCSgewFzRxyi7iEmucxFfgnOE4FBuBx5Wkm2LYq+gRJqr5BBbroxCiMFSwj1DVhxWwKUjcdeWvxaPjOTvgBGN/Y2MHh/2kVHgP4nOWEp9Chel0jXIeKVcjWUWtRl+s862d4ZmEHbAYqElPC1BLUwjdxPvYWWLqt5kkqXuvpoHL7zIN20HbDtNAEou/shFPrC2WwTRw72EzT5b35JSj3tKG4OJpIz/AxLcIfU4MjKJGIUCdcImUUH/HBoHjdjA1X9li6ukL9n7/MZBEqRCa2uYRTqLk6ZCcK7+XmDwhyEoMyqgdgYL6+EkOBFnhG2rC/7SGqy7z0NlzfYA671aTt48DYXvb9t22mZ5ciKHSGToG23+sbxyEu7c0slneuj69h6dxjk9gp9BCiMs1K2K8B9SD0I/iSWBA8pNe3vdx8pMtkBqgkKuZLD2U/pgLxyMif3WmJySSFAAOA8+D/aUHrWEcNQt0V/ZRmwkkttjOvnkOcLZ95FcD8+07bA18EAA+P8fJ5NJHgMlCIIClm2j8az1//pL0gdw5LJOanc7Kzr0Juy1bFlXLEWY51u/XEu06E+ROnwtlvvh0v78SSlmkzz+IhRIcMayFAT0U7oX0OLhO7Qg2CDJDYgfIZ8hRWbz16dkx7N/guXkuPSaMPIO653QsrTdhRX5y00uFlQZlQSxlsSBWI+qohNDKKj8NfH/qY+oRRhCP1wD6jH2Le6daN86muLOjs9MtofypZxdc+mASJO35Wttyvy6rwnj3OXRaEPUouRZdHcmbNGwi95ZnO+Gvv0A8+IUaAiRaIKo9/x5MABo4sHI/SB6UtvK1361B3XG4yZXPDpDddTAsYnp/EYIEizAxc5Br2NcJbKqXpqMTeq4HSG6IDXdUOiu/RzuWPt0slellzcJ1dfBw+XviRBAtK7wqt0U5HWEVz55FWMzK4/+oAIvbcaZZ4rZOuTazlFYBlu8unM1jDhCFOrbBBWXCQZCEF/EWcIx68ZM7THpPwEhi4rhPFWumBUqzISYValnel3V5djJ6TGOIE7q+xLpK/Ah1ePVyd+CdEEWj691jGFiuS44lu1xspslsJei9lDp0WIkld63zSlf/nvpspk/GjzgUohDMaKCQ21S+ZeoJQAHLaFw9AShN92nogGd1b8I2sn7kz1PFdo5Oqv/Gg1U6Ox+s0lnnOFEEwTFzFzHCTwlIm8vdzsI6+JEXL9H+j7S9JVN2Z50H6HpblasVmeBXBeaU5p3u6JZ6og+QzRbwU9bYyz/F2jlWIQa2th4ptDf5ePOxOjMvSSaMk6FpThKXEoDhTgEYrhIeV8+GoY81teMnlWpu5ICByQY4a7j3WRzlLBc8p2OPZbh5vnmPtSXMin2DhOeZm4BYt5mLTly/P/f3WYIelGKU1+l13JqEsMNBPIktRtZXc+6GT70FC0zYrvS7JNuOe0q34+DbVlLeC8+HeYXohRTR+zuJ3v9uoutmAKQxbfpcqGd0l9ZNf4D0hf9k0hdTpB/sJP3gTaRPauDTcSAyoJ/gwIXH1LY+I78YN6FB8MJzpvas61m6bcudmhZRXnlQ1zNeMka8a83ZxcJ4qXGa6EHuSGpM4lLEFYjWvJ32+9KN9+4ru+y26HpLn5Inc/GbBt+/oj7RZo5rucYSXTkEKSbIGPj9sQ942M604SxstL7+7GbomMt9++s/RInv8//49ZcPn3RXi6jf+DnrE6wvP92CVMepFJ/R/5m36NOHTx//J+YJeBgyeAF+8PFz1gw+ffySbH8fP6PzcxT6EYEhtPvNTU9UrxSlc9Vr0Hj8GgbA4E8f8x8/f93VYbzZ4V//+vw1l/vnP9nPdJInSNOv9NnR7EvynOpvOfpUbfjy/wAAAP//AwCFJ/UJ91sAAA==
  recorded_at: Tue, 14 May 2019 14:00:00 GMT
recorded_with: VCR 6.0.0
