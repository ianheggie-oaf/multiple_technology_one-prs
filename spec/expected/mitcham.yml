---
- council_reference: '080/0159/21'
  address: 45 Dinwoodie Avenue CLARENCE GARDENS SA 5039
  description: Attached Carport - Clarence Gardens Kindergarten
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0159%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-05-03'
- council_reference: '080/0166/21'
  address: 86 Highfield Drive CRAIGBURN FARM SA 5051
  description: Attached Verandah
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0166%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-21'
- council_reference: '080/0308/21'
  address: 5 Nunyara Crescent BELAIR SA 5052
  description: Dwelling Alterations and Additions, Domestic Outbuilding & Deck
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0308%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-27'
- council_reference: '080/0332/21'
  address: 65 Symon Crescent CRAIGBURN FARM SA 5051
  description: Detached Dwelling
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0332%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-22'
- council_reference: '080/0335/21'
  address: 48A East Parkway COLONEL LIGHT GARDENS SA 5041
  description: Dwelling Addition
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0335%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-22'
- council_reference: '080/0356/21'
  address: 1 Pembroke Place COLONEL LIGHT GARDENS SA 5041
  description: Solar Panels
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0356%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-20'
- council_reference: '080/0361/21'
  address: 65/5 Mount Barker Road URRBRAE SA 5064
  description: Retrospective Dwelling alteration (Window in Masonry wall)
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0361%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-20'
- council_reference: '080/0364/21'
  address: 28 The Grove LOWER MITCHAM SA 5062
  description: Combined Fencing & Retaining Walls with a Height Greater than 2m, Front
    Fence with Masonry Walls & Pillars
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0364%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-22'
- council_reference: '080/0373/21'
  address: 33 Salisbury Crescent COLONEL LIGHT GARDENS SA 5041
  description: Fencing
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0373%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-20'
- council_reference: '080/0402/21'
  address: 73 Hillcrest Drive EDEN HILLS SA 5050
  description: Dependent Relative Accommodation with Attached Raised Deck
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0402%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-20'
- council_reference: '080/0413/21'
  address: 1-2/216 Daws Road DAW PARK SA 5041
  description: Crown Development - Hammond Care Daw Park - Specialist Dementia Care
    Facility
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0413%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-05-12'
- council_reference: '080/0416/21'
  address: Belair Recreation Park BELAIR SA 5052
  description: Section 49 - Re-development of Playford Lake Precinct including Construction
    of Access Arrangements, associated Infrastructure & Buildings - Upper Sturt Road
    Belair National Park
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0416%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-27'
- council_reference: '080/0417/21'
  address: 505 Fullarton Road NETHERBY SA 5062
  description: SECTION 49 - Impacts to State Heritage items resulting from the proposed
    upgrade to the intersection of Cross Road & Fullarton Road (Urrbrae House Gatehouse
    & Urrbrae House)
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F0417%2F21&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-05-03'
- council_reference: '080/1166/20.01'
  address: 16 Carrick Hill Drive SPRINGFIELD SA 5062
  description: Vary DA 080/1166/20 for Alterations and Additions, Verandah & Swimming
    Pool & Safety Barrier - to Remove Existing Stone Outbuilding, Extend Dining Room
    to Incorporate Lounge & Change Pool Location
  info_url: https://com-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=080%2F1166%2F20.01&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2021-04-16'
